src/commands/build/wranglerjs/mod.rs

```diff
@@ -159,35 +159,53 @@ fn setup_build(target: &Target) -> Result<(Command, PathBuf, Bundle), failure::E
 
     command.arg(format!("--wasm-binding={}", bundle.get_wasm_binding()));
 
-    let webpack_config_path = PathBuf::from(
-        &target
-            .webpack_config
-            .clone()
-            .unwrap_or_else(|| "webpack.config.js".to_string()),
-    );
+    let webpack_config_path = if let Some(webpack_config) = &target.webpack_config {
+        Some(PathBuf::from(webpack_config.clone()))
+    } else if target.site.is_none() {
+        message::warn("In Wrangler v1.6.0, you will need to include a webpack_config field in your wrangler.toml to build with a custom webpack configuration.");
+        Some(PathBuf::from("webpack.config.js".to_string()))
+    } else {
```

Comment:
this conditional sequence took me a while to grok; from what i understand, the final `else` option is for the case when a site is specified and there is no webpack_config, so no webpack config is used? I'd love a comment to clarify that :D

Reply:
cool